new-module/vaults/README.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Angle _New_ Module agTokens Vaults
+---
+
+# agTokens Vaults
+
+The core mechanism of this module relies on a vault system. Users can deposit collateral in vault contracts, and borrow a certain amount of agTokens from this vault as a debt that will have to be repaid later. By doing so, they can keep their exposure to the tokens deposited as collateral, while being able to spend the borrowed funds. They can also use this mechanism to increase their exposure to the collateral they own, on-chain and in one transaction. 
+
+**TLDR**
+* Borrow agTokens from tokens deposited as collateral in the protocol
+* Leverage collateral exposure in one transaction
+* Transfer your debt between vaults to avoid liquidation
+
+
+## Main features
+
+The main features of vaults are Borrow & Repay agTokens, and leverage collateral exposure. 
+
+### Borrow & Repay agTokens
```

Comment:
Let's take examples from Abracadabra docs and add some examples here

Reply:
Like I put x of collateral get y of stablecoins ...

Reply:
We also need to list all the actions possible when you own a vault: add collateral, remove collat, borrow, repayDebt and so on. Many other things and examples could be added to make it more intelligible

We should say that it has been optimized for gas and for composability and that anything can be done in just one transaction

Reply:
We could add a short example, but most of this should belong in the user guide IMO